blocks/v2-inpage-navigation/v2-inpage-navigation.css

```diff
@@ -0,0 +1,208 @@
+  display: none;
+}
+
+.v2-inpage-navigation__item button,
+.v2-inpage-navigation__selected-item-wrapper {
+  background: none;
+  border: 0;
+  color: var(--c-primary-black);
+  cursor: pointer;
+  display: block;
+  font-family: var(--ff-body-bold);
+  font-size: var(--body-2-font-size);
+  line-height: var(--body-2-line-height);
+  margin: 0;
+  padding: 14px 24px;
+  width: 100%;
+}
+
+.v2-inpage-navigation__item button:hover,
+.v2-inpage-navigation__item button:focus,
+.v2-inpage-navigation__selected-item-wrapper:hover,
+.v2-inpage-navigation__selected-item-wrapper:focus {
+  background-color: #F1F1F1;
+}
+
+.v2-inpage-navigation__item button:active,
+.v2-inpage-navigation__selected-item-wrapper:active {
+  background-color: #F1F1F1;
+}
+
+/* stylelint-disable-next-line no-descending-specificity */
+.v2-inpage-navigation__item button {
+  max-width: none;
+  text-align: left;
+  width: 100%;
+}
+
+/* stylelint-disable-next-line no-descending-specificity */
+.v2-inpage-navigation__selected-item-wrapper {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.v2-inpage-navigation__selected-item-wrapper svg {
+  --color-icon: var(--c-accent-red);
+
+  height: 16px;
+  transition: transform var(--duration-small) var(--easing-standard);
+  width: 16px;
+}
+
+/* Customization when dropdown is open */
+.v2-inpage-navigation__dropdown--open .v2-inpage-navigation__items {
+  display: block;
+}
+
+.v2-inpage-navigation__dropdown--open .v2-inpage-navigation__selected-item-wrapper {
+  background-color: #F1F1F1;
+}
+
+.v2-inpage-navigation__dropdown--open .v2-inpage-navigation__selected-item-wrapper svg {
+  transform: rotate(180deg);
+}
+
+/* END Customization when dropdown is open */
+
+/* Red button */
+.v2-inpage-navigation__cta:any-link {
+  align-items: center;
+  background-color: var(--button-primary-red-enabled);
+  color: var(--c-primary-white);
+  display: flex;
+  font-family: var(--ff-body);
+  font-size: 14px;
+  font-style: normal;
+  font-weight: 500;
+  letter-spacing: 1.12px;
+  line-height: 18px;
+  padding: 0 20px;
+  text-decoration: none;
+}
+
+.v2-inpage-navigation__cta:hover,
+.v2-inpage-navigation__cta:focus {
+  background-color: var(--button-primary-red-hover);
+}
+
+.v2-inpage-navigation__cta:active {
+  background-color: var(--button-primary-red-pressed);
+}
+
+.v2-inpage-navigation__cta--desktop {
+  display: none;
+}
+
+@media (min-width: 1200px) {
+  :root {
+    --inpage-navigation: 96px;
+  }
+
+  .v2-inpage-navigation__wrapper {
+    align-items: center;
+    gap: 24px;
+    max-width: var(--wrapper-width);
+    padding: 24px 0;
+  }
+
+  .v2-inpage-navigation__selected-item-wrapper {
+    display: none;
+  }
+
+  .v2-inpage-navigation__items,
+  .v2-inpage-navigation__dropdown--open .v2-inpage-navigation__items {
+    display: flex;
+  }
+
+  /* stylelint-disable-next-line no-descending-specificity */
+  .v2-inpage-navigation__items {
+    box-shadow: none;
+    gap: 24px;
```

Comment:
since you have `justify-content `rule, you can remove the `gap` property

Reply:
This gap is to maintain a minimum distance btw items in case there are a lot of items. So I'm not removing it.